pennylane/io.py

```diff
@@ -407,6 +407,61 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`~.NoiseModel`.
+
+    Args:
+        noise_model (qiskit_aer.noise.NoiseModel): a Qiskit ``NoiseModel`` instance
+        kwargs: Optional keyword arguments for conversion of the noise model.
+
+    Keyword Arguments:
+        verbose (bool): show a complete list of Kraus matrices for ``qml.QubitChannel`` instead of
+            number of Kraus matrices and the number of qubits they act on. The default is ``False``
+        decimal_places (int): number of decimal places to round the Kraus matrices when they are
+            being displayed for each ``qml.QubitChannel`` with ``verbose=False``.
+
+    Returns:
+        qml.NoiseModel: An equivalent noise model constructed in PennyLane
```

Comment:
Consider using this for consistency: A PennyLane :class:`~.NoiseModel` instance

Reply:
I've made the statement consistent with the other `from_qiskit` functions. Also, as the return type already captures the information regarding the `class`, I avoid mentioning it again in the return description to avoid redundency.